storage/src/main/java/tech/pegasys/artemis/storage/client/RecentChainData.java

```diff
@@ -128,6 +122,12 @@ public Store getStore() {
 
   // NETWORKING RELATED INFORMATION METHODS:
 
+  public void updateBestBlock(ReadOnlyStore store) {
+    Bytes32 headBlockRoot = store.getFinalizedCheckpoint().getRoot();
+    BeaconBlock headBlock = store.getBlock(headBlockRoot);
+    updateBestBlock(headBlockRoot, headBlock.getSlot());
+  }
```

Comment:
I don't think this is right.  It works for genesis because there are no unknown blocks, but starting up after genesis we need to run fork choice to select the best block.